docgen/generate-docs.js

```diff
@@ -125,11 +127,13 @@ function addTypeAliasLinks(data) {
    * Select .tsd-signature-type because all potential external
    * links will have this identifier.
    */
-  const fileTags = htmlDom.window.document.querySelectorAll(".tsd-signature-type");
-  fileTags.forEach(tag => {
+  const fileTags = htmlDom.window.document.querySelectorAll(
+    '.tsd-signature-type'
+  );
+  fileTags.forEach((tag) => {
```

Comment:
Nit: Google style guide prohibits `forEach` since it has no benefits over `for of`

Reply:
Ah, this wasn't a line I touched just Prettier fixing up style, but easy enough to fix. Done for all instances in the file.